components/TweetList.js

```diff
@@ -1,29 +1,95 @@
-export default function TweetList({ tweets }) {
-  // { fields: { tweetUrl: '', tags: ['']} }
+import React from 'react'
+import styled from 'styled-components'
+
+const Heading = styled('div')`
+  font-size: 2.5rem;
+  text-align: center;
+  padding: 1rem;
+`
+
+const Container = styled('div')`
+  text-align: center;
 
-  const uniqueTags = tweets.reduce(() => {})
-  // ['SEO', 'Link Tracking', ...]
+  button {
+    color: black;
+    padding: 8px 16px;
+    text-decoration: none;
+    transition: background-color 0.3s;
+    border: 1px solid #ddd;
+    margin: 0 4px;
+    cursor: pointer;
+    border-radius: 5px;
+    border-radius: 5px;
+
+    :hover {
+      background-color: #ddd;
+    }
+
+    :focus {
+      outline: 0;
+    }
+  }
+  .activeTag {
+    background-color: #ddd;
+  }
+`
+const Tweet = styled('div')`
+  padding: 0.5rem;
+`
+
+export default function TweetList({ tweets }) {
+  const uniqueTags = []
+  tweets.map(tweet => {
+    for (let i = 0; i < tweet.fields.tags.length; i++) {
+      if (uniqueTags.indexOf(tweet.fields.tags[i]) === -1) {
```

Comment:
Since this is a flat array of string, we can do this instead:

```suggestion
      if (!uniqueTags.includes(tweet.fields.tags[i])) {
```